test/deploy/rackhd_stack_init.py

```diff
@@ -230,36 +230,33 @@ def test11_add_management_server(self):
         print "**** Creating management server."
         usr = ""
         pwd = ""
-        # find correct BMC passwords from global config
+        # find correct BMC passwords from credentials list
         for creds in fit_common.fitcreds()['bmc']:
```

Comment:
Cleared my previous comment.  bmc is a credential array not a dictionary.  Leave code as.